### ISSUE TITLE
Does not work

### ISSUE THREAD

**Issue by rbiggs** [octref/polacode#87]
This plugin only creates an image with default console logs. There doesn't seem to be any way to have it capture and image of selected text. Is there some undocumented key combination to capture an image??? The docs do not explain anything.

**Comment by jens1o** [comment]
You need to copy and paste the content out of the code window you want to have as an image.

**Comment by Rietty** [comment]
On top of that you need to click the shutter button to save the image.

**Comment by VAggrippino** [comment]
This doesn't work for me, either. Clicking the round button that looks kinda like a camera shutter doesn't do anything.

**Comment by gpgabrielpaiva** [comment]
![image](https://user-images.githubusercontent.com/1836254/52648809-435fc000-2ece-11e9-9c37-eccdb2874cc1.png)

I have this problem on click.

**Comment by nahtnam** [comment]
Doesnt work for me either.

**Comment by morgoth990** [comment]
Same here, paste dont work

**Comment by Springle3152** [comment]
Doesn't work for me either. I get this message:
Extension 'pnp.polacode' uses the 'vscode.previewHtml' command which is deprecated and will be removed soon. Please file an issue against this extension to update to use VS Code's webview API.

**Comment by pariola** [comment]
same issue here

**Comment by macropp** [comment]
I get this on Insiders version:

```
[2019-05-07 13:15:10.336] [renderer1] [error] command 'vscode.previewHtml' not found: Error: command 'vscode.previewHtml' not found
    at Proxy._tryExecuteCommand (file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:3014:836)
    at r.then.i (file:///Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:3014:717)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

**Comment by PLASMAchicken** [comment]
This should have been fixed in #105

**Comment by Rietty** [comment]
Is the copy to clipboard still stopped by vscode?

**Comment by octref** [comment]
Fixed via #105.
